src/providers/address/address.ts

```diff
@@ -116,22 +81,22 @@ export class AddressProvider {
   }
 
   public extractAddress(address: string): string {
-    let extractedAddress = address
+    const extractedAddress = address
       .replace(/^(bitcoincash:|bchtest:|bitcoin:)/i, '')
       .replace(/\?.*/, '');
     return extractedAddress || address;
   }
 
   public isValid(address: string): boolean {
```

Comment:
isValid's argument's name should be 'str' or similar. not address, to avoid confusion.

Reply:
Done